Silicon/CommonSocPkg/Include/Library/PlatformInfo.h

```diff
@@ -30,4 +30,29 @@ GetMaxCpuPciePortNum (
   VOID
 );
 
+/**
+  Return CPU Sku
+
+  @param[in]  UINT32             CpuFamilyModel
+  @param[in]  UINT16             CpuDid
+
+  @retval     UINT8              CPU Sku
+**/
+UINT8
+GetCpuSkuInfo (
+  IN UINT32 CpuFamilyModel,
+  IN UINT16 CpuDid
+);
+
+/**
+  Return CPU Sku
+
+  @retval UINT8              CPU Sku
+**/
+UINT8
+EFIAPI
+GetCpuSku (
+  VOID
+ );
```

Comment:
It looks the implementation on these 2 functions are commented out. and no consumer to these 2 functions in this patch. so we don't need add them, right?

Reply:
we need them, as these are used in the code. This patch is associated with a common lib patch that was merged last week.

Reply:
Removed this code for now. will add it in right place if required.